diff --git a/dist/codegen.d.ts b/dist/codegen.d.ts
index b1c56efa136747a80e7c624d18cd60b17eaa3f55..27745f686c84c62ef16e99acc7d7f1d02e43837d 100644
--- a/dist/codegen.d.ts
+++ b/dist/codegen.d.ts
@@ -1,7 +1,6 @@
 import type { Dialect } from 'kysely';
-import { type GenerateOptions } from 'kysely-codegen';
 export declare class Codegen {
     dialect: Dialect;
     constructor(dialect: Dialect);
-    generate(opts: Omit<GenerateOptions, 'dialect'>): Promise<string>;
+    generate(opts: any): Promise<string>;
 }
diff --git a/dist/introspector.d.ts b/dist/introspector.d.ts
index ce06b353388eb64cfa1cb8ce3471ea1f235acd54..09fb98dd5d25a43bebd9ad975bf101cf903e6fa2 100644
--- a/dist/introspector.d.ts
+++ b/dist/introspector.d.ts
@@ -1,7 +1,6 @@
 import { Kysely } from 'kysely';
-import { DatabaseMetadata, Introspector, type ConnectOptions, type IntrospectOptions } from 'kysely-codegen';
-export declare class KyselyPGliteIntrospector extends Introspector<any> {
-    connect(options: ConnectOptions): Promise<Kysely<any>>;
-    protected getTables(options: IntrospectOptions<any>): Promise<import("kysely").TableMetadata[]>;
-    introspect(options: IntrospectOptions<any>): Promise<DatabaseMetadata>;
+export declare class KyselyPGliteIntrospector {
+    connect(options: any): Promise<Kysely<any>>;
+    protected getTables(options: any): Promise<import("kysely").TableMetadata[]>;
+    introspect(options: any): Promise<any>;
 }
